tools/snp.sh

```diff
@@ -1148,6 +1279,10 @@ main() {
         SKIP_IMAGE_CREATE=true
         shift; shift
         ;;
+      -s|--svsm)
+        SVSM=false
```

Comment:
This is why all your logic seems to be backwards. If people are passing the svsm flag why are you making the variable false?

Reply:
Yes, you got it finally.

Reply:
it shouldn't be that way, its confusing...

Reply:
I agree. This needs to switch, otherwise the logic is backward and doesn't make any sense. If the flag is passed, then the flag is enabled, not disabled.